rosette-scala/src/main/scala/coop/rchain/rosette/PC.scala

```diff
@@ -1,10 +1,10 @@
 package coop.rchain.rosette
 
-case class PC(relative: Int) {
+case class PC(relative: Int, absolute: Int) {
```

Comment:
In the C++ code, `absolute` is a pointer to the particular instruction being executed, while `relative` is the index of that instruction.  I don't think we need `absolute`.  If we decide that we need it, we should revisit OpJmp* where I used `relative` instead.